sample-yamls/pod_cyclictest.yaml

```diff
@@ -35,6 +35,9 @@ spec:
       value: "1000"
     - name: delay
       value: "0"
+    - name: TRACE_THRESHOLD
+      value: ""
+      # TRACE_THRESHOLD requires privileged=true and debug volume
```

Comment:
Please add the volumeMount and volume for the debug volume (commented out). See pod_oslat.yaml for details.

Reply:
Added in latest commit